spec/physical_field_spec.rb

```diff
@@ -0,0 +1,66 @@
+RSpec.describe SparqlToSwSolr::InstanceSolrDoc::PhysicalField do
+
+  let(:instance_uri) { 'http://ld4p-test.stanford.edu/1234567890#Instance' }
+  let(:isd) { SparqlToSwSolr::InstanceSolrDoc.new(instance_uri) }
+  let(:sparql_conn) { double('sparql client') }
+  let(:solutions) { RDF::Query::Solutions.new }
+
+  before(:each) do
+    allow(isd).to receive(:sparql).and_return(sparql_conn)
+    allow(sparql_conn).to receive(:query).and_return(solutions)
+  end
+
+  context '#physical_values' do
+    it 'is a String consisting of the results from a query solution' do
+      solutions << RDF::Query::Solution.new(noteType: 'Physical details',
+                                            extentLabel_1: '15 p.',
+                                            noteLabel: 'ill.',
+                                            instanceDimensions_3: '5 x 5 in.')
+      physical_values = isd.send(:physical_values)
+      expect(physical_values).to eq '15 p. : ill. ; 5 x 5 in.'
```

Comment:
just a thought: you could combine lines to get `expect(isd.send(:physical_values)).to eq 'blah'` if you wanted.